tests/core/test_test.py

```diff
@@ -129,16 +148,21 @@ async def test_action_unlikely_intent_1(
         """
     )
 
+    # We train on the above story so that RulePolicy can memorize
+    # it and we don't have to worry about other actions being
+    # predicted correctly.
+    agent = await _train_rule_based_agent(moodbot_domain, file_name)
```

Comment:
Out of curiosity: What didn't work with the monkey patching?

Reply:
The monkey patching is for triggering `action_unlikely_intent` at a specific instance. All other actions still need to be predicted correctly which is what this rule based agent would be trained to do.

Reply:
ooooh, I see 👍🏻

Reply:
@wochinge Do you have any other comments on this or can I merge it?

Reply:
Sorry, I was waiting for an answer to the second comment but just had another look and makes sense 👍🏻

Reply:
Ahh, my bad too. I could have updated that one too. 🙈